source/projects/next.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Next
+repo: zeit/next.js
+homepage: https://learnnextjs.com/
+language: JavaScript
+license: MIT
+templates: JavaScript
+description: A framework for statically-exported React apps
+startertemplaterepo: zeit/next.js/tree/master/examples
```

Comment:
This has to point to a standalone repo that generates a static site starter for Next.js. If there's not an official example repo available, just remove this field.

Reply:
@erquhart I removed it ❌